### PR TITLE
Pull #1: Desktop version of the Portfolio project 

### PR DESCRIPTION
**Pull request by Ismailco**
# changes:
>### add navbar menu
>### create media query

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -372,3 +377,232 @@ hr {
   background-color: #2230d2;
   border-color: #2230d2;
 }
+
+@media all and (min-width: 1200px) {
```
Comment (Dipeshtwis) [inline styles/style.css]: - [x] Use media queries for the Desktop breakpoint in the CSS file.(min-width > 992px - 1100px). 

you can use like this

```
@media screen and (min-width: 992px) {
/*
.
.
*/
}

```

File: index.html [hunk 2]
```diff
@@ -190,66 +218,65 @@ <h2 class="connect-link"><a href="#">Let's connect</a></h2>
           </li>
         </ul>
         <button type="button" class="work-btn">Get my resume</button>
```
Comment (Dipeshtwis) [inline index.html]: - [x] You have already made this as a `button` element so no need to write explicitly with type `button` instead you can give `type= "submit"` or with no type attribute (note that it defaults to a submit type when no type attribute is used). Please make changes to all the places.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Ismailco , 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Good use of Flexbox and Grid properties.
👍 Good to see that you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/style.css [hunk 3]
```diff
@@ -372,3 +377,232 @@ hr {
...
+
+@media screen and (min-width: 992px) {
+  .header {
+    align-items: baseline;
+  }
+
+  .logo {
+    margin: 25px 155px;
+  }
+
```
Comment (MahmoudBakr23) [inline styles/style.css:389]: - [ ] **OPTIONAL**
For changing the look of the website on the medium-sized screen you could use another media query this way
```
@media screen and (min-width: 365px) and (max-width: 991px) {
//
}
```
Screenshot

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Ismailco 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**